File: samples/tagging/python-individual/upload-audio-file.py
import requests
from requests.auth import HTTPBasicAuth

api_key = ""    # Your API Key
audio_file = "" # Your audio file path
url = "https://api-us.musiio.com/api/v1/upload/file"

files = [
  ('audio', open(audio_file,'rb'))
]

response = requests.post(url, auth=HTTPBasicAuth(api_key, ''), files = files)

print(response.text.encode('utf8'))

File: samples/js/upload-audio-file.js
// fileInput is the input element on your webpage
/* HTML
<input type="file" id="fileInput">
<script>
    var fileInput = document.getElementById("fileInput");
</script>
*/

const API_KEY = ""; // API Key here
var myHeaders = new Headers();
myHeaders.append("Authorization", "Basic " + btoa(API_KEY + ":"));

var formdata = new FormData();
formdata.append("audio", fileInput.files[0], "file");

var requestOptions = {
  method: 'POST',
  headers: myHeaders,
  body: formdata,
  redirect: 'follow'
};

fetch("https://api-us.musiio.com/api/v1/upload/file", requestOptions)
  .then(response => response.text())
  .then(result => console.log(result))
  .catch(error => console.log('error', error));